### PR TITLE
GSOC/Outreachy wrap up post. 

### LINKED ISSUE #249: GSoC completion blog post
Once GSoC is completed for our students, summarize their projects and link to their final blog posts.

### PR DESCRIPTION
**Pull request by hmacdope**
fixes #249 

The final post for GSOC and Outreachy studentships 2022.

### CODE REVIEW

File: _posts/2022-11-02-gsoc2022-final.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "GSoC / Outreachy 2022 wrap-up"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
```
Comment (orbeckst) [inline _posts/2022-11-02-gsoc2022-final.md:8]: Does outreachy have a logo that we can use?

Comment (hmacdope) [reply _posts/2022-11-02-gsoc2022-final.md:8]: This would probably do the trick. 

https://commons.wikimedia.org/wiki/File:Outreachy-logo.svg

**Comment by hmacdope** [comment]
@BFedder @aya9aladdin @umak1106 quick bump how is everyone progressing with their blog posts? Would be good to tie this off soon-ish

**Comment by aya9aladdin** [comment]
> @BFedder @aya9aladdin @umak1106 quick bump how is everyone progressing with their blog posts? Would be good to tie this off soon-ish

Sorry for being late, I was in a vacation last week. I'm finishing it by today or the latest by tomorrow

**Comment by BFedder** [comment]
Sorry, busy few weeks - I'll have mine ready by the end of this weekend.

**Comment by umak1106** [comment]
I'll have mine ready by the end of this week too ..

File: _posts/2022-11-02-gsoc2022-final.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+title="Google Summer of Code / Outreachy 2022" alt="Google Summer of Code / Outreachy 2022"
+style="float: right; height: 12em; " />
+</p>
+
+This year MDAnalysis participated in two student programs with wonderful students from around the globe. 
+We continued our participation in [Google Summer of Code][gsoc] for another year and also participated in [Outreachy][outreachy]! for the first time.
```
Comment (orbeckst) [inline _posts/2022-11-02-gsoc2022-final.md:14]: Do we need to mention NumFOCUS here — did they pay for Outreachy?

File: _posts/2022-11-02-gsoc2022-final.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+layout: post
+title: "GSoC / Outreachy 2022 wrap-up"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
+title="Google Summer of Code / Outreachy 2022" alt="Google Summer of Code / Outreachy 2022"
+style="float: right; height: 12em; " />
+</p>
```
Comment (orbeckst) [inline _posts/2022-11-02-gsoc2022-final.md:11]: Add Outreachy logo.

File: _posts/2022-11-02-gsoc2022-final.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+This year MDAnalysis participated in two student programs with wonderful students from around the globe. 
+We continued our participation in [Google Summer of Code][gsoc] for another year and also participated in [Outreachy][outreachy]! for the first time. 
+Huge thanks to these organisations for supporting MDAnalysis and the students.
+
+## Google Summer of Code
+ 
+We had[two GSOC projects][gsoc-projects] this year who made wonderful contributions.  None of this would have been possible without our students: [Aya Mohamed Alaa][project-aya] (@aya9aladdin), and [Bjarne Feddersen][project-bjarne] (@BFedder).
```
Comment (orbeckst) [inline _posts/2022-11-02-gsoc2022-final.md:19]:
```suggestion
Our two outstanding GSoC contributors [Aya Mohamed Alaa][project-aya] (@aya9aladdin), and [Bjarne Feddersen][project-bjarne] (@BFedder) successfully completed their [GSOC projects][gsoc-projects].
```

- they are "contributors"
- condense prose a little bit

File: _posts/2022-11-02-gsoc2022-final.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+You can read more about their projects and packages in their blog posts:
+
+* Aya :  
+* Bjarne: 
+
+—  GSOC Mentors @jbarnoud @hmacdope @ojeda-e @IAlibay @fiona-naughton @orbeckst @lilyminium @richardjgowers
+
+## Outreachy 
+
+** Outreachy People fill in the details **
```
Comment (orbeckst) [inline _posts/2022-11-02-gsoc2022-final.md:30]: @jbarnoud @micaela-matta @richardjgowers please add something short about Outreachy

File: _posts/2022-11-02-gsoc2022-final.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (orbeckst) [inline _posts/2022-11-02-gsoc2022-final.md:31]:
```suggestion

* Uma:

```
Keep the style similar to GSOC.

**Review by orbeckst — CHANGES_REQUESTED** [review]

File: _posts/2022-11-02-gsoc2022-final.md (hunk 2 above)
Comment (hmacdope) [reply _posts/2022-11-02-gsoc2022-final.md]: They are like a low-tier sponsor of outreachy, I'm not sure we need to mention them  in particular?

Comment (richardjgowers) [reply _posts/2022-11-02-gsoc2022-final.md]: I think CZI paid for our outreachy participation iirc

File: _posts/2022-11-02-gsoc2022-final.md (hunk 5 above)
Comment (orbeckst) [reply _posts/2022-11-02-gsoc2022-final.md:39]: ping @jbarnoud @micaela-matta @richardjgowers — we need some prose in the blog post

**Comment by orbeckst** [comment]
@hmacdope sorry, I think I messed up your branch. You might have to rebase against the latest main. I don't want to do it and force-push.

**Comment by jbarnoud** [comment]
If we have individual blog posts, do we need this one?

**Comment by hmacdope** [comment]
Just waiting on #262 and then ill hammer this into shape.

**Comment by orbeckst** [comment]
All student posts are up. Could someone update the summary post, change the date, and merge, please?

**Comment by hmacdope** [comment]
@MDAnalysis/coredevs this should be good to go.

**Review by orbeckst — APPROVED** [review]
great, thanks for shepherding it to completion @hmacdope ! You're a ⭐ .

Thank you to @umak1106 @aya9aladdin @BFedder , you're awesome. Well done!

**Pull request merged**